### ISSUE TITLE
Support custom landing pages & logos

### ISSUE THREAD

**Issue by thomashoneyman** [rnons/purescript-halogen-storybook#2]
Hi @rnons -- thanks for your work on this library! It makes it dead-simple to create a showcase for examples and I'm using it in [formless](https://github.com/thomashoneyman/purescript-halogen-formless), among others.

Two limitations for me include:

- You can't customize the landing page. It just has a smattering of text and "powered by halogen-storybook" whereas I'd prefer to be able to put information about the library there.
- You can't set the logo text / put your own logo at the top-left. I'd rather be able to have the library name shown up at the top.

Of course, I think your library ought to be highlighted and linked to from the storybook! But those two places seem less desirable than a prominent footer link.

What do you think of a structure more like the image below?

- The user can write their own app / library name at the top left
- halogen-storybook is linked at the bottom left in the 'footer'
- (not shown) the user can then customize the landing page without removing the only link to Storybook

<img width="925" alt="x" src="https://user-images.githubusercontent.com/10245104/42415390-6d3d65b4-8201-11e8-857d-1823514cba16.png">

**Comment by rnons** [comment]
Thanks for let me know this library is being used.

I have been thinking about allowing user to override the index page and logo for a while, only haven't put it into code. 

For the index page, I'm thinking about using a special key (e.g. empty string) for index page, like

```
stories = Object.fromFoldable
  [ Tuple "" $ proxy ExpIndex.component
  ]
```

To support logo, I need to change `runStorybook` to take a record

```
runStorybook
  { stories
  , logo :: HH.PlainHTML
  }
```

Does this make sense to you? I will put up something later today.

**Comment by thomashoneyman** [comment]
That works for me, thanks!

**Comment by rnons** [comment]
@thomashoneyman released as v0.4.0, please see if it works.

**Comment by thomashoneyman** [comment]
Works just fine, thanks!
